protocol/harness.ts

```diff
@@ -1,17 +1,34 @@
 /// @dev The Fuel testing harness.
 /// A set of useful helper methods for testing Fuel.
-import { ethers } from 'hardhat';
+import { ethers, upgrades } from 'hardhat';
 import { BigNumber as BN, Signer } from 'ethers';
 import { FuelSidechainConsensus } from '../typechain/FuelSidechainConsensus.d';
 import { FuelMessagePortal } from '../typechain/FuelMessagePortal.d';
 import { L1ERC20Gateway } from '../typechain/L1ERC20Gateway.d';
 import { Token } from '../typechain/Token.d';
-import { BinaryMerkleTree } from '../typechain/BinaryMerkleTree.d';
 import { computeAddress, SigningKey } from 'ethers/lib/utils';
 
-// This is the Harness Object.
+// Well known private key for testing.
+export const DEFAULT_POA_KEY = '0xc87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3';
+
+// All deployable contracts.
+export interface DeployedContracts {
+	fuelMessagePortal: FuelMessagePortal;
```

Comment:
Is the indentation in this file messed up? It looks like it might be 2 tabs worth

Reply:
The formatter seems to be configured to use tabs, which GitHub is showing as a very big indent. This was not changed in the current PR. I'll include fixing this along with some of the other nitpicks in a separate PR (like renaming a few things for consistency)